swarm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.4
+
+WORKDIR /go/src/github.com/docker
+RUN git clone https://github.com/deis/swarm
+WORKDIR /go/src/github.com/docker/swarm
+RUN git fetch origin nodefailover
+RUN git checkout nodefailover
+ENV GOPATH /go/src/github.com/docker/swarm/Godeps/_workspace:$GOPATH
+RUN CGO_ENABLED=0 go install -v -a -tags netgo -ldflags "-w -X github.com/docker/swarm/version.GITCOMMIT `git rev-parse --short HEAD`"
```

Comment:
IMO we should be using the same build pipeline that we use for logger, logspout, builder etc. where the binary is built locally and then injected into the Docker image.